core/src/main/java/org/elasticsearch/index/query/TemplateQueryParser.java

```diff
@@ -67,28 +51,18 @@ public TemplateQueryParser(ScriptService scriptService) {
      * values. Handles both submitting the template as part of the request as
      * well as referencing only the template name.
      *
-     * @param context
+     * @param parseContext
```

Comment:
If you already touching the JavaDoc and renaming the parameter - why not add some documentation to it as well?